src/rules/__tests__/no-test-callback.test.ts

```diff
@@ -16,41 +17,132 @@ ruleTester.run('no-test-callback', rule, {
     'test("something", function() {})',
     'test("something", async function () {})',
     'test("something", someArg)',
+    'test("something", (...args) => {args[0]();})',
```

Comment:
we should error here as well, although we can't apply the fix.

Jest just provides the one argument, so it's just the fixer that doesn't work, with the rule active we should still warn about this case